eslint/babel-eslint-plugin-internal/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@babel/eslint-plugin-internal",
```

Comment:
Nit: I'd prefer to call this `@babel/eslint-plugin-development-internal`, since it's more similar to `@babel/eslint-plugin-development` than to `@babel/eslint-plugin`.